README.md

```diff
@@ -1,5 +1,71 @@
 # molgenis-ui-filter
 
+## Filter Container
+
+The filter container takes a descriptive input array to create a set of filters. It will also require a 2 way bound output object to receive the filters state
+
+```
+<FilterContainer
+  v-model="selections"
+  :filters="filters"
+/>
+```
+
+### Filter input object specifications
+
+```
+filters: [ {
+    name: 'search',
+    label: 'Search',
+    description: 'search by name',
+    collapsed: true,
+    placeholder: 'test',
+    type: SearchFilterVueObject
+}, {
+    name: 'country',
+    label: 'Countries',
+    collapsed: false,
+    options: [{ value: 'nl', text: 'Nederland' }, { value: 'de', text: 'Duitsland' }, { value: 'be', text: 'België' }],
+    type: CheckboxVueObject
+}]
+```
+
+Every filter will have the following global properties:
+
+| name        | required | default value | description | 
+|-------------|----------|---------------|-------------|
+| name        | true     | nvt           | string: name and id of the filter |
+| label       | false    |               | string: Name of the filter as used by the UI |
+| description | false    |               | Descriptive text that is used by the UI |
+| collapsed   | false    | true          | boolean: Open/Closed state of the card that holds the filter |
```

Comment:
If we want a prop to control the state, we'll also need to fire an event when it changes. But I'd rather first describe what we have